clients/da-vinci-client/src/main/java/com/linkedin/davinci/config/VeniceServerConfig.java

```diff
@@ -578,8 +582,11 @@ public VeniceServerConfig(VeniceProperties serverProperties, Map<String, Map<Str
         serverProperties.getInt(MAX_LEADER_FOLLOWER_STATE_TRANSITION_THREAD_NUMBER, 20);
 
     blobTransferManagerEnabled = serverProperties.getBoolean(BLOB_TRANSFER_MANAGER_ENABLED, false);
-    snapshotRetentionTimeInMin = serverProperties.getInt(BLOB_TRANSFER_SNAPSHOT_RETENTION_TIME_IN_MIN, 30);
+    snapshotRetentionTimeInMin = serverProperties.getInt(BLOB_TRANSFER_SNAPSHOT_RETENTION_TIME_IN_MIN, 60);
     maxConcurrentSnapshotUser = serverProperties.getInt(BLOB_TRANSFER_MAX_CONCURRENT_SNAPSHOT_USER, 5);
+    blobTransferMaxTimeoutInMin = serverProperties.getInt(BLOB_TRANSFER_MAX_TIMEOUT_IN_MIN, 60);
+    blobTransferDisabledOffsetLagThreshold =
```

Comment:
This one is hard to config properly as the different stores would configure different kinds of ready-to-serve threshold.
As the VT would contain the heart-beat message, so can we configure a time-based threshold instead?

Reply:
My concern is, today we don't persist the HB we have consumed up to in OffsetRecord, so if we don't subscribe and ingestion we don't know the HB lag we are at... We can persist that in OffsetRecord in theory, but that would be another change and protocol evolution involved..

Reply:
Can we consume VT backward  from the persisted checkpoint to figure out the latest HB timestamp?